python/src/gz/sim/Sensor.hh

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
```

Comment:
```suggestion
 * Copyright (C) 2023 Open Source Robotics Foundation
```

Reply:
Done.

Reply:
I still see 2021

Reply:
Weird I might have changed in other place and forgot about this one. It should be done now.